var/spack/repos/builtin/packages/py-annexremote/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyAnnexremote(PythonPackage):
+    """git annex special remotes made easy."""
+
+    homepage = "https://github.com/Lykos153/AnnexRemote"
+    pypi     = "annexremote/annexremote-1.5.0.tar.gz"
+
+    version('1.5.0', sha256='92f32b6f5461cbaeefe0c60b32f9c1e0c1dbe4e57b8ee425affb56f4060f64ef')
+
+    depends_on('python@3:', type=('build', 'run'))
```

Comment:
You can remove this line, this package also supports Python 2.

Reply:
You are right, they will drop Python 2 not until the next release.